backend/src/common/enums/api/users-api-path.enum.ts

```diff
@@ -1,5 +1,6 @@
 enum UsersApiPath {
   ROOT = '/',
+  IDPARAM = '/:id'
```

Comment:
```suggestion
  $ID = '/:id'
```
Any params should be with the `$` prefix